.meteorignore

```diff
@@ -0,0 +1,3 @@
+/testapp
+/tests
```

Comment:
meteor seems to include the tests directory and it's contents even if it is included in `.meteorignore`, probably because it's a special directory.